src/main/host/process.c

```diff
@@ -606,9 +606,23 @@ static void _process_start(Process* proc) {
     worker_setActiveThread(NULL);
 
     if (proc->pause_for_debugging) {
-        fprintf(stderr, "Managed process '%s' has started with PID %d. Press ENTER to continue...",
+        // will block until logger output has been flushed
+        // there is a race condition where other threads may log between the fprintf() and raise()
+        // below, but it should be rare
+        logger_flush(logger_getDefault());
+
+        // print with hopefully a single syscall to avoid splitting these messages (fprintf should
+        // not buffer stderr output)
+        fprintf(stderr,
+                "** Pausing with SIGTSTP to enable debugger attachment to managed process '%s' "
+                "(pid %d)\n"
+                "** If running under Bash, resume Shadow by pressing Ctrl-Z to background this "
```

Comment:
Ah darn, I was hoping bash would already treat the child process getting SIGSTOP the way it does ctrl-z. Oh well.

Reply:
Yeah I don't think there's a way to background the process from the process itself.